pkg/compare/compare_test.go

```diff
@@ -662,3 +662,230 @@ metadata:
 		})
 	}
 }
+
+func TestYAMLCmpWithYamlInline(t *testing.T) {
+	tests := []struct {
+		desc string
+		a    string
+		b    string
+		want interface{}
+	}{
+		{
+			desc: "ConfigMap data identical",
```

Comment:
you could probably omit this test, since the reordered test below is a superset.

Reply:
Sure, removed.